handbook/company.md

```diff
@@ -225,12 +225,12 @@ The Fleet product is released every three weeks. By syncing the whole company to
 
 
 ### Why use agile methodology?
-Releasing software iteratively gets changes and improvements into the hands of users faster and generally results in software that works. This makes contributors fitter, happier, and more productive. See [the agile manifesto](https://agilemanifesto.org/). for more information.
+Releasing software iteratively gets changes and improvements into the hands of users faster and generally results in software that works. This makes contributors fitter, happier, and more productive. See [the agile manifesto](https://agilemanifesto.org/). For more information.
```

Comment:
This edit doesn't make it better.  It was def wrong before too.  I think the period needs removed.